pages/_en-US/ds-index/special-games.md

```diff
@@ -310,3 +310,37 @@ Currently, nds-bootstrap does not have Slot-2 emulation. Included are some possi
 - Personal Trainer: Walking -> Mii Channel
 - Ultimate Band -> Ultimate Band (Wii)
 - WarioWare D.I.Y. -> WarioWare D.I.Y. Showcase
+
+### Other (Carts that have features that don't fall into an above category) 
+- Learn with Pokemon: Typing Adventures 
+   - Cartridge has special bluetooth chip for connecting to the keyboard
+- Pokemon HG/SS
+   - Cartridge Has IR port for connecting to the PokeWalker
+- ElecktroPlankton
+   - Cartridge has no save RAM
+- DS Download station
+   - Series of cartridges used to distribue demo games
+- Sonic Classics Collection
+   - Genesis emulation structure has been converted into an emulator used in TWiLight Menu++
+- Nitro Development Cart
+   - Has extra ram for development of games for the Nintendo DSi
+- Personal Trainer: Walking
+   - Has twin IR based pedometer
+- Slide Adventure MAGKID
+   - Comes packaged with a mouse like slot 2 peripheral
+- Daigasso! Band Brothers
+   - Had a slot 2 cartridge that unlocked more songs
+- Arkanoid
+   - Came bundled with a slot 2 paddle controller
+-  Astroarts Hoshizora Navi 
+   - Has elongated cartridge with built in "compass" orientation sensor
+- Play-Yan 
+   - A slot 2 MP3 player 
+- Face Training
+   - Came packaged with a slot 2 camera
+- My Weightloss trainer 
+   - Bundled with a slot 2 pedometer
+- Beyblade Series
+   - Bundled with an adapter for connecting to a Beypointer
+- Nintendo DS Tv tuner
+   - Analog tv tuner for the DS 
```

Comment:
It's a digital TV tuner, not analog fyi. Specifically it can do [1seg](https://en.wikipedia.org/wiki/1seg) which is the broadcast specifically for mobile devices in Japan

Reply:
Oops